### DISCUSSION TITLE
18🈲

### DISCUSSION THREAD

**Discussion by Tartaglia770** [share121/inter-knot#3267] [其他]
![mmexport1724131583086](https://github.com/user-attachments/assets/7f27f378-9b60-464f-a6aa-bd96e0de8cff)
家人们趁现在没有人举报赶紧<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by lllliufeng** [comment]
狠活，举办了（

**Comment by Larksonguesinaspic** [comment]
666

**Comment by RaphaelHider** [comment]
我寻思谁敢举办这个

**Comment by xingyuehao** [comment]
这个委托我接了（bushi）<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by sane520** [comment]
是正经东西吗，不正经，正经我都不看
![Uploading subtitle_2024_0131_191446_edit_553741381751441.png…]()

**Comment by anbi-demala** [comment]
太6了<img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/d364cbf606b633be67a1536289347556_8458169816302857767.png" alt="时光交响之章表情包10" width="75" >
